nc-games-fe/src/Components/Vote-On-Review.jsx

```diff
@@ -0,0 +1,29 @@
+import React, {useState, useEffect} from 'react';
+import { updateReviewVotes } from './utils';
+
+const VoteButtons = ({ reviewId, setVotes }) => {
+    const [voteUnSuccessful, setVoteUnSuccessful] = useState(false)
+
+    const handleVote = (e) => {
+        setVotes((currentValue) => currentValue + 1 )
+        e.target.disabled= true;
+        updateReviewVotes(reviewId)
+        .then(() => {
+            setVoteUnSuccessful(false)
+        })
+        .catch((err) => {
+            setVoteUnSuccessful(true)
+            e.target.disabled = false
+                  })
+      };
+
+        return (
+        <div>
+            <button onClick={handleVote} >Upvote</button>
+            {voteUnSuccessful && <p>Your vote was unsuccessful! Please refresh the page to try again.</p>}
```

Comment:
Nice little error message here. Not a trick question but does the user really need to refresh? With React we should avoid needing to refresh.... your logic for an error right now is: show an error message, and you re-enable the vote button (and I recommend reducing the vote number back by -1 using more optimistic rendering), hopefully this is good enough for the user to allow them to try again, as opposed to them having to refresh?